packages/components/input/stories/textarea.stories.tsx

```diff
@@ -302,3 +302,16 @@ export const IsInvalid = {
     errorMessage: "Please enter a valid description",
   },
 };
+
+export const Clearable = {
+  render: Template,
+
+  args: {
+    ...defaultProps,
+    placeholder: "Enter your description",
+    defaultValue: "junior@nextui.org",
+    // isClearable: true,
```

Comment:
why is this line commented out?

Reply:
This is just to test for no other effects. I'll delete it！